samples/requirement-targeting-mutual-dependency/nodejs/README.md

```diff
@@ -0,0 +1,170 @@
+---
+page_type: sample
+products:
+- office-365
+languages:
+- javascript
+title: Specify relationships between app capabilities
+description: Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet)
+extensions:
+  contentType: samples
+  createdDate: 08/20/2024 10:02:21 PM
+urlFragment: officedev-microsoft-teams-samples-requirement-targeting-mutual-dependency-nodejs
+---
+
+# Microsoft Teams RT Sample App Feature.
+
+- Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet)
```

Comment:
1. Should highlight that this sample app is about mutual dependency, not one way. So probably along the lines of "...which specifies mutual-dependency relationship between app capabilities..."
2. Also shouldn't be "Microsoft Teams", but "Microsoft 365".

Reply:
fixed